### ISSUE TITLE
Filters Modified Date Interval Wording

### ISSUE THREAD

**Issue by JonoSuave** [microsoft-search/pnp-modern-search#1287]
**Version used**
4.2.3

**Question**
Is there a way to make Date Interval template display all items over the past three months? For example, filter on the managed property LastModifiedTime and select Past 3 months. In reality it isn't the past three months, it's everything older than one month and less than three months. For the regular business user this is confusing. Anyway to have the Past 3 months option include the the past month as well? If not, how can I edit the text of Past 3 months text for users to better understand?

**Screenshots**
![Screen Shot 2021-09-13 at 11 28 46 AM](https://user-images.githubusercontent.com/22969971/133129704-e3f7393d-72b0-48d0-afc8-2f7889ae81e0.png)

**Comment by wobba** [comment]
I agree and don't remember the exact reasoning for this. It should for all `Past` options start with now and subtract the interval. Older than a year, starts one year back from now.

**Comment by JonoSuave** [comment]
Is the text of the date interval options editable?

**Comment by wobba** [comment]
No, and we should show the static text imo.

**Comment by FranckyC** [comment]
Hi @JonoSuave if you look at the code, intervals are built excluding each others, meaning the start of an interval is the end of the previous one:

![image](https://user-images.githubusercontent.com/5659122/133150039-7b6ab58f-e83a-4d55-9ff1-51621cb4cc7b.png)

We could simply always start from Date.now() to include past intervals as well. Let me make some tests and let you know.

**Comment by JonoSuave** [comment]
Awesome, yeah that does appear at first glance like a simple update.

**Comment by FranckyC** [comment]
@JonoSuave It appears it is not simple as it looks :(. The reason is we used fixed buckets for the refimement query meaning these intervals will always apply.

`Created(discretize=manual/${pastYear}/${past3Months}/${pastMonth}/${pastWeek}/${past24hours}/${today})`

Then we only show values with count above 0 in filters. If we change the behavior to only use exactly the interval defined by the filter value when clicked (ex: from 'today' to last 3 months, including past 24 hours, past week and past month), we "loose" the ability to choose other intervals to refine the query afterward (as there is only this interval available):

`Created(discretize=manual/${past3Months}/${today})`

![image](https://user-images.githubusercontent.com/5659122/133326190-3df85241-0084-4131-98ed-bac2b70cfd14.png)

Not sure this is the best option in terms of UI. Why not renaming the values by their true meaning like "From past month to past 3 months"? This would actually make more sense according to the response made by the search API (which does basically the same with dates).

**Comment by FranckyC** [comment]
@JonoSuave your thoughts about this solution?

**Comment by JonoSuave** [comment]
Yeah, bummer the buckets are exclusive. That being said, I like the verbiage update. If the verbiage is too long perhaps something like: 
- From last 1 - 7 days
- From last 8 - 30 days
- From last 1-3 months
- From last 3 - 12 months

**Comment by wobba** [comment]
OOB UX has these intervals where the "Past" ones are from today - X days. And should be possible to use the buckets with the logic we want.

![image](https://user-images.githubusercontent.com/51104/134508301-794463e8-f673-4f07-ae38-474aff6cef6f.png)
